src/limb.rs

```diff
@@ -60,6 +58,8 @@ pub(crate) const HI_BIT: usize = Limb::BITS - 1;
 
 /// Big integers are represented as an array of smaller CPU word-size integers
 /// called "limbs".
+#[allow(renamed_and_removed_lints)]
+#[allow(clippy::derive_hash_xor_eq)] // renamed to `derived_hash_with_manual_eq` since 1.69
```

Comment:
Perhaps just bump clippy?

Reply:
I'm not sure what's your policy on CI, currently clippy is run on 1.65 there.

Reply:
I mean, if you want to keep running clippy on MSRV, that's what we have to do.

Reply:
It doesn't have to be MSRV. We can bump it to 1.72.

It's just pinned so it doesn't randomly break every time there's a new release.

Reply:
Ok, I'll bump it